### PR TITLE
#722: Supporting optional Stock for Article

### PR DESCRIPTION
**Pull request by hani-ibrahim**
fixing #722

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/CheckoutSummaryArticleSelectCollectionView.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ enum CheckoutSummaryArticleRefineType {
     func count(selectedArticle: SelectedArticle) -> Int {
         switch self {
         case .size: return selectedArticle.article.availableUnits.count
-        case .quantity: return selectedArticle.unit?.stock ?? 0
+        case .quantity: return min(selectedArticle.unit?.stock ?? 1, 10)
```
Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutSummaryArticleSelectCollectionView.swift]: Magic number should be avoided. How about

```swift
enum StockQuantity: Int {
  case one = 1, two, three, four, five, six, seven, eight, nine, ten
}
```

SCNR.

**Comment by bonkey** [comment]
👍

File: AtlasUI/AtlasUI/CheckoutSummaryArticleSelectCollectionView.swift (hunk 1 above)
Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/CheckoutSummaryArticleSelectCollectionView.swift]: 🙄

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Pull request merged**